### ISSUE TITLE
`mvn install` fails due to authorization error

### ISSUE THREAD

**Issue by yanne** [DP-3T/dp3t-sdk-backend#1]
Running `mvn install` fails with:

`[ERROR] Failed to execute goal on project dpppt-backend-sdk-model: Could not resolve dependencies for project org.dpppt:dpppt-backend-sdk-model:jar:1.0.0-SNAPSHOT: Failed to collect dependencies at ch.ubique.openapi:springboot-swagger-3:jar:1.1.2: Failed to read artifact descriptor for ch.ubique.openapi:doc-annotations:jar:0.0.3: Could not transfer artifact ch.ubique.openapi:doc-annotations:pom:0.0.3 from/to github (https://maven.pkg.github.com/Ubique-OSS/springboot-swagger3): Not authorized -> [Help 1]`

**Comment by ubamrein** [comment]
Sorry for that, we are new to github packages and forgot to mention how to actually load packages. 

But please have a look at [Github Documentation](https://help.github.com/en/packages/using-github-packages-with-your-projects-ecosystem/configuring-apache-maven-for-use-with-github-packages) to have some information. 

TL;DR You need to generate a Github Token on your account under [Developer Settings](https://github.com/settings/tokens), which has at least "read:packages" privileges.

Please let us know if that fixes your issue.

**Comment by yanne** [comment]
Thanks, I was new to github repositories as well, but got it working now. 
Have to do the whole configuration with repositories and profiles though.

**Comment by ubamrein** [comment]
I will close this issue now. I added some information on which packages are hosted on github and added the link to the github help. I also gave some hints on how to opt-out from code which uses our Github packages.

Anyways, thanks for reporting!
